sphinx_rtd_theme/layout.html

```diff
@@ -207,6 +207,20 @@
   </script>
   {% endif %}
 
+  {% if theme_analytics_id %}
+  <!-- Theme Analytics -->
+  <script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', '{{ theme_analytics_id }}', 'auto');
+  ga('send', 'pageview');
+  </script>
+
+  {% endif %}
```

Comment:
So what would be the proper solution? Seems like we just wouldn't set this on RTD, and the theme can support it if it wants?